main/src/main/scala/sbt/internal/TaskProgress.scala

```diff
@@ -86,20 +86,24 @@ private[sbt] final class TaskProgress(currentRef: ProjectRef) extends ExecutePro
 
   private[this] val console = ConsoleOut.systemOut
   private[this] def readyLog(): Unit = {
-    console.println("")
-    console.println("")
-    console.println("")
-    console.println("")
-    console.println("")
-    console.println("")
-    console.print(CursorUp5)
+    if (isLogReady.get) ()
+    else {
+      console.println("")
```

Comment:
This is a bit inscrutable. Maybe you could add a comment that explains why you need to print these blank lines followed by CursorUp5. I'd also find this slightly more readable if it were: `console.print(s"${"\n" * 5}$CursorUp5")`.

Reply:
I initially added this to make space on the bottom for task progress. I am going to see if we still need this actually.